package/ci/circleci.yml

```
@@ -349,6 +353,7 @@ jobs:
     - install-cmake:
         version: "3.5.2"
     - install-basis
```

YAML is a mess, right?

```suggestion
    - install-basis:
```

Same in the other cases below, that'll make it parse correctly.